src/Raven.Server/Documents/Handlers/Processors/TimeSeries/AbstractTimeSeriesHandlerProcessorForGetDebugSegmentsSummary.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Threading.Tasks;
+using JetBrains.Annotations;
+using Raven.Server.Web;
+using Sparrow.Json;
+
+namespace Raven.Server.Documents.Handlers.Processors.TimeSeries
+{
+    internal abstract class AbstractTimeSeriesHandlerProcessorForGetDebugSegmentsSummary<TRequestHandler, TOperationContext> : AbstractTimeSeriesHandlerProcessor<TRequestHandler, TOperationContext>
```

Comment:
this does not have to inherit from AbstractTimeSeriesHandlerProcessor
This looks like a proxy read processor?